imageio/plugins/ffmpeg.py

```diff
@@ -512,7 +514,8 @@ def _load_infos(self):
             if match is not None:
                 hms = map(float, line[match.start()+1:match.end()].split(':'))
                 self._meta['duration'] = duration = cvsecs(*hms)
-                self._meta['nframes'] = int(round(duration*fps))
+                if fps:
+                    self._meta['nframes'] = int(round(duration*fps))
```

Comment:
I wish I wasn't so swamped at work. I want to look into pyav to see how it handles these things. I'm curious if it has any different approaches or results. But I'm not sure when I'll get time. But when I do I'll let you know. I did play with pyav a little. It seem d like it's current seeking may not be as efficient as imageios seek with ffmpeg. It seemed like they just read every frame until getting to the requested frame.